dockers/docker-snmp/snmp_yml_to_configdb.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+import os
+import sys 
+
+import yaml
+
+from sonic_py_common.logger import Logger
+from swsssdk import ConfigDBConnector
+
+db = ConfigDBConnector()
+db.connect()
+
+SYSLOG_IDENTIFIER = 'snmp_yml_to_configdb.py'
+logger = Logger(SYSLOG_IDENTIFIER)
+logger.set_min_log_priority_info()
+
+snmp_comm_config_db = db.get_table('SNMP_COMMUNITY')
+snmp_config_db_communities = snmp_comm_config_db.keys()
+snmp_general_config_db = db.get_table('SNMP')
+snmp_general_keys = snmp_general_config_db.keys()
+
+full_snmp_comm_list = ['snmp_rocommunity', 'snmp_rocommunities', 'snmp_rwcommunity', 'snmp_rwcommunities']
+
+
+if not os.path.exists('/etc/sonic/snmp.yml'):
+    logger.log_info('/etc/sonic/snmp.yml does not exist')
+    sys.exit(1)
+
+with open('/etc/sonic/snmp.yml', 'r') as yaml_file:
+    yaml_snmp_info = yaml.load(yaml_file)
```

Comment:
Python 3 will print deprecation warning if `yaml.load` is called without a `Loader=` argument. Please add `Loader=yaml.FullLoader` to this function call.

Reply:
fixed.